config/routes.rb

```diff
@@ -5,4 +5,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get 'task', to: 'app/task#index'
```

Comment:
What do you think if we use double quotes?
```suggestion
  get "task", to: "app/task#index"
```